view/theme/vier/config.php

```diff
@@ -9,6 +9,9 @@ function theme_content(&$a){
 	if(!local_user())
 		return;
 
+	if (!function_exists('get_vier_config'))
+		return;
```

Comment:
I guess we really have to think it over. Problem is that you can't apply changes in the admin interface for "Vier" if you hadn't selected it as your theme.

This is a fundamental problem but I don't have a quick solution for it. We could file an issue for that so that we don't forget it. (But this shouldn't prevent this pull request from being merged)